### PR TITLE
[14.0][MIG] stock_putaway_product_template: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Please, @pedrobaeza and @sergio-teruel can you review it?

@Tecnativa TT31847

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review

File: stock_putaway_product_template/models/stock_putaway_rule.py [hunk 1]
```diff
@@ -19,16 +19,15 @@ class StockPutawayRule(models.Model):
     @api.depends("product_id")
     def _compute_product_tmpl_id(self):
         for rec in self:
-            rec.product_tmpl_id = False
```
Comment (pedrobaeza) [inline stock_putaway_product_template/models/stock_putaway_rule.py]: There's a path where you don't assign any value, so I still see this as needed.

Comment (victoralmau) [reply stock_putaway_product_template/models/stock_putaway_rule.py]: If I'm not mistaken v14 already defines a default value in the registers with no value in a compute method and therefore takes care of this. In views it works fine if that's what you mean.

Comment (pedrobaeza) [reply stock_putaway_product_template/models/stock_putaway_rule.py]: OK, I see that is a computed writeable, so it's OK then.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**